theforeman.org/pipelines/release/source/smart-proxy.groovy

```diff
@@ -36,10 +36,10 @@ pipeline {
             steps {
                 dir(project_name) {
                     git url: git_url, branch: git_ref
-                }
-                script {
-                    archive_git_hash()
-                    sourcefile_paths = generate_sourcefiles(project_name: project_name, source_type: source_type)
```

Comment:
We still need this line. Here is what it should look like:

```
                 dir(project_name) {
                     git url: git_url, branch: git_ref

                    script {
                        archive_git_hash()
                    }
                 }

                 script {
                     sourcefile_paths = generate_sourcefiles(project_name: project_name, source_type: source_type)
                 }
```

Reply:
Okay, sorry I got little confuse about it. I though we can do it at the same place.